docker/runtime/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #-----------------------------    BUILD STAGE   ------------------------------#
 ###############################################################################
 
-FROM codecompass:dev as builder
+FROM modelcpp/codecompass:dev as builder
```

Comment:
Why changing this to the DockerHub tag name? If someone is building the runtime images, I think that he/she would build the dev image, too.

Reply:
This ensures that the official runtime images is always built from the official developer image, and can not be mixups with local containers while building.

Reply:
That could raise issues, because if a user would like to build custom images, then building the runtime image would not depend on the locally built dev image. Also, with this modification the walkthrough in `docker/README.md` would not work.

Reply:
That is fair. I'm reverting this.